ebml/EbmlCrc32.h

```diff
@@ -58,7 +58,7 @@ DECLARE_EBML_BINARY(EbmlCrc32)
     }
 
     void AddElementCRC32(EbmlElement &ElementToCRC);
-    bool CheckElementCRC32(EbmlElement &ElementToCRC);
```

Comment:
ABI break. Same as all the other instances of adding `const`. Not that I'm against it; just saying we'll have to be careful with bumping the version number accordingly (according to semver this would require bumping the major component, → 2.0.0) & communicating the break to distribution packagers.